### PR TITLE
small UI adjustments

### PR DESCRIPTION
**Pull request by jbwilson8**
[OTTER-47](https://openstax.atlassian.net/browse/OTTER-47)

[OTTER-47]: https://openstax.atlassian.net/browse/OTTER-47?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/researcher/study/[encodedStudyId]/edit/form.tsx [hunk 1]
```diff
@@ -57,32 +57,56 @@ export const Form: React.FC<{ studyId: string; study: FormValues }> = ({ studyId
...
+                        name="irbDocument"
+                        control={control}
+                        readOnly
+                    />
+                    {/* <Text bd="1px solid #ccc" p="5px" bg="#ddd" className={inputStyle} data-testid="irb-doc">
                         {' '}
                         IRB Document.pdf
-                    </Text>
-                    <Text fs="italic" c="dimmed" w="10%">
+                    </Text> */}
```
Comment (nathanstitt) [inline src/app/researcher/study/[encodedStudyId]/edit/form.tsx:92]: should commented out code be removed?

File: src/app/researcher/study/request/[memberIdentifier]/page.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ export default async function MemberHome({ params }: { params: { memberIdentifie
 
                 <List>
                     {/* <ListItem>Submit proposal for review</ListItem> */}
```
Comment (nathanstitt) [inline src/app/researcher/study/request/[memberIdentifier]/page.tsx:28]: this isn't something you added, but we could remove this commented out ListItem as well

**Review by nathanstitt — APPROVED** [review]
looks great, just a few commented out lines that could be removed

**Review by therealmarv — APPROVED** [review]
Same remarks as Nathan about commented out code. Maybe remove comments unless it will be soon important. Otherwise everything okay.

**Pull request merged**